### PR TITLE
Added definition of GPSACP message element

### PR DESCRIPTION
**Pull request by nm-mrt**
As requested by Tobias I highjack this repository to discuss the definition of GPSACP which is not fully defined by WTP - SCG Reference Points

### CODE REVIEW

**Comment by thz** [comment]
This is actually not hijacking but specifying the details of the to-be-implemented feature.

File: docs/GPSACP.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+**Type** 15
+
+**Length** >= 5
+
+**Timestamp** The WTP’s time when the meassurment was recorded
+
+**WWAN Id** WWAN Interface Id, used to differentiate between multiple WWAN interfaces, MUST be between
+one (1) and 16.
+
+**GPSATC** The full output (including the starting $GPSATC) of the AT$GPSATC command the WWAN Interface:
```
Comment (RoadRunnr) [inline docs/GPSACP.md]: should be `GPSACP`

File: docs/GPSACP.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+```
+$GPSACP:<UTC>,<latitude>,<longitude>,<hdop>,<altitude>,<fix>,<cog>,<spkm>,<spkn>,<date>,<nsat> 
+
```
Comment (RoadRunnr) [inline docs/GPSACP.md]: The format is support by several m2m LTE moduls. It my not be the original, but this format was taken from Telit LTE modules with GPS support (e.g.: http://www.telit.com/fileadmin/user_upload/products/Downloads/3G/xe910/Telit_HE910_A-GPS_Application_Note_r4.pdf)

File: docs/GPSACP.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<UTC>: HHMMSS
+  HH: Stunde (00..23)
+  MM: Minute (00..59)
+  SS: Sekunde (00..60) (Ja - 60 - Schaltsekunden...)
+<latitude>: ddmm.mmmmD
+  dd: Grad (00..90)
+  mm.mmmm: Minuten mit dezimalen Nachkommerstellen (00.0000 .. 59.9999)
```
Comment (RoadRunnr) [inline docs/GPSACP.md]: this should be in English, please.

**Pull request merged**